src/biogeochem/CNVegCarbonFluxType.F90

```diff
@@ -3512,6 +3535,16 @@ subroutine RestartBulkOnly ( this, bounds, ncid, flag )
             long_name='grain C to food', units='gC/m2/s', &
             interpinic_flag='interp', readvar=readvar, data=this%grainc_to_food_patch)
 
+       call restartvar(ncid=ncid, flag=flag,  varname='leafc_to_biofuelc', xtype=ncd_double,  &
+            dim1name='pft', &
+            long_name='leaf C to biofuel C', units='gC/m2/s', &
+            interpinic_flag='interp', readvar=readvar, data=this%leafc_to_biofuelc_patch)
+
+	   call restartvar(ncid=ncid, flag=flag,  varname='livestemc_to_biofuelc', xtype=ncd_double,  &
```

Comment:
Please fix indentation of this line (see comment elsewhere about tabs vs. spaces). (But this is, of course, unnecessary if you remove these restart variables, as I suggest in another comment.)